### PR TITLE
Feat: User model and RESTful API / CSRF

### PR DESCRIPTION
**Pull request by TLB-MISS**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (Zagoshipda) [inline .vscode/settings.json:1]: Are we going to include vscode settings on our project?

File: backend/probloom/prob/views.py [hunk 2]
```diff
@@ -1,59 +1,107 @@
...
 from .models import User, UserStatistics
 
 # Create your views here.
-@ensure_csrf_cookie
-def userList(request):
-    if request.method == 'GET':
-        userAllList = [user for user in User.objects.all().values()]
-        return JsonResponse(userAllList, safe=False)
-    elif request.method == 'POST':
+def signup(request):
```
Comment (Zagoshipda) [inline backend/probloom/prob/views.py:15]: error priority required from sprint 3 master @clarkindy

Comment (Zagoshipda) [inline backend/probloom/prob/views.py:15]: error priority required from sprint 3 master @clarkindy

File: frontend/probloom/src/containers/SignUp/SignUp.tsx [hunk 3]
```diff
@@ -39,11 +39,7 @@ class SignUp extends Component<Props, State> {
     };
   }
 
-  componentDidMount() {
-    this.props.onGetAllUsers();
-  }
-
-  onClickSignUpButton = () => {
+  onClickSignUpButton = async () => {
```
Comment (Zagoshipda) [inline frontend/probloom/src/containers/SignUp/SignUp.tsx:42]: email, username, password valid check required from sprint 3 master @clarkindy

File: .vscode/settings.json (hunk 1 above)
Comment (TLB-MISS) [reply .vscode/settings.json:1]: It is not necessary. I don't know why .gitignore didn't work.

File: backend/probloom/prob/models.py [hunk 4]
```diff
@@ -1,17 +1,14 @@
...
-    logged_in = models.BooleanField(default=False)
+class User(AbstractUser):
+    pass
 
-    def __str__(self):
-        return self.username
 
 class UserStatistics(models.Model):
     lastActiveDays = models.IntegerField(default=0)
+    #user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="statistics")
```
[the remark this answers is not in the document]
Comment (TLB-MISS) [reply backend/probloom/prob/models.py:11]: It's something I added arbitrarily for convenience. No backend for profile has been implemented at all yet. Please implement and pull.

**Pull request closed without merging**